### PR TITLE
status: show RO/RW status

### LINKED ISSUE #835: `tt status`: show RO/RW status
It is very useful piece of information.

For example, we expect that one instance is RW and all the other ones are RO. If all the instances are RO, it means that something unexpected occurred.

### PR DESCRIPTION
**Pull request by 0x501D**
`tt status` now displays the mode of the instance.

Closes #835

### CODE REVIEW

File: cli/status/status.go [hunk 1]
```diff
@@ -39,6 +40,23 @@ func Status(runningCtx running.RunningCtx) error {
 		if procStatus.Code == process_utils.ProcessRunningCode {
 			fmt.Fprintf(tw, "%d", procStatus.PID)
 		}
+
+		conn, err := connector.Connect(connector.ConnectOpts{
```
Comment (psergee) [inline cli/status/status.go:44]: Can we set the timeout for the connection explicitly? To 1 second for example. This way, the status won't take too long to print.

Comment (0x501D) [reply cli/status/status.go:44]: Timeout is hardcoded in `func Connect(opts ConnectOpts) (Connector, error)`:
```go
greetingConn.SetReadDeadline(time.Now().Add(greetingOperationTimeout))
greetingOperationTimeout = 3 * time.Second
```

**Comment by psergee** [comment]
The table is not aligned:
```
$ tt status app
INSTANCE           STATUS          PID     MODE
app:s2-replica     NOT RUNNING     
app:stateboard     RUNNING         151914     RW
app:router         NOT RUNNING     
app:s1-master      NOT RUNNING     
app:s1-replica     NOT RUNNING     
app:s2-master      NOT RUNNING
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Changed
+
+- `tt status`: displays the mode of the instance.
```
Comment (psergee) [inline CHANGELOG.md:12]: pls add info about new `tt status` cli option.

File: cli/status/status.go [hunk 3]
```diff
@@ -1,65 +1,69 @@
...
-	colorBytesNumber = 9
-
-	// padding is the padding between two columns of the table.
-	padding = 5
-)
-
-var header = []string{"INSTANCE", "STATUS", "PID"}
+// StatusOpts contains options for tt status.
+type StatusOpts struct {
+	Pretty bool
```
Comment (psergee) [inline cli/status/status.go:15]: Add comment for external member.

**Review by psergee — APPROVED** [review]
Thank you for the patch. Lgtm.

File: CHANGELOG.md (hunk 2 above)
Comment (0x501D) [reply CHANGELOG.md:12]: Added

File: cli/status/status.go (hunk 3 above)
Comment (0x501D) [reply cli/status/status.go:16]: Added

**Pull request merged**